cmd/influxd/config.go

```diff
@@ -251,6 +248,71 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Address       string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          int    `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeperator string `toml:"name-seperator"`
+}
+
+// Default carbon port per http://graphite.readthedocs.org/en/1.0/feeding-carbon.html
+const defaultGrahitePort = 2004
+
+// TCPAddr returns the TCP address to connect on.
+// If port is not specified in the config file, it will default to defaultGraphitePort (2004)
+// If address is not specified,it will default to the defaultBindAddress passed in
+func (g Graphite) TCPAddr(defaultBindAddress string) *net.TCPAddr {
```

Comment:
I know we've been beating on this function, so sorry if it's getting tiresome, but this implementation is a little inconsistent. I would either push or defaults in, or none. In other words, I think the function signature might be better if:
```
TCPAddr(defaultBind, defaultPort)
```
Not a big deal, just noticed it in passing.